trade-finance-manager-api/src/v1/api-response-types/fee-record-correction-request-review-response-body.ts

```diff
@@ -0,0 +1,13 @@
+import { ReportPeriod, SessionBank } from '@ukef/dtfs2-common';
+
+export type FeeRecordCorrectionRequestReviewResponseBody = {
+  bank: SessionBank;
+  reportPeriod: ReportPeriod;
+  correctionRequestDetails: {
+    facilityId: string;
+    exporter: string;
+    reasons: string[];
```

Comment:
Should we type this as `RecordCorrectionReason[]` from `libs/common/src/types/record-correction-reason.ts` ?

Reply:
Ah yes, not sure why I didn't! 😅